jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointCaller.java

```diff
@@ -46,8 +46,8 @@
 class RegistryEndpointCaller<T> {
 
   /**
-   * <a
-   * href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308">https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308</a>.
+   * <a href =
```

Comment:
@chanseokoh Is this the correct styling of a-href ?

Reply:
Hmm... let's keep your change if the formatter keeps changing this. As long as builds succeed, we should be good.